app/src/main/java/invoice_automation/module/QuickBooksModule.java

```diff
@@ -103,7 +104,12 @@ public Map<InvoiceType, Invoice> getOrCreateInvoicesFromRegistration(@NonNull Re
      * @param invoice The Invoice to send
      */
     public void sendInvoice(@NonNull Invoice invoice) {
-        // TODO: IA-9
+        try {
+            this.dataService.sendEmail(invoice, invoice.getBillEmail().getAddress());
+        } catch (FMSException e) {
+            throw new QuickBooksException("Exception sending invoice", e);
+        }
+        invoice.setEmailStatus(EmailStatusEnum.EMAIL_SENT);
```

Comment:
I'm curious, is this line needed?

Reply:
Yeah, you need to change the status manually, sendEmail doesn't alter it